### PR TITLE
Upgrade prettier

### PR DESCRIPTION
**Pull request by jason-codaio**
#### Is this adding or improving a _feature_ or fixing a _bug_?

Update to latest prettier. Also updates code with `yarn lint --fix` to match new package.

#### What's the new behavior?

More idiomatic code style

#### Have you checked that...?

- [X] The new code matches the existing patterns and styles.
- [X] The tests pass with `yarn test`.
- [X] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [X] The relevant examples still work. (Run examples with `yarn start`.)

#### Does this fix any issues or need any specific reviewers?

Reviewers: @timbuckley

### CODE REVIEW

**Comment by timbuckley** [comment]
How about just adding `"arrowParens": "avoid"` to `.prettierrc`?

**Comment by jason-krypton** [comment]
@timbuckley I suppose thats up to you. I can update if that is the preference.

**Comment by jason-krypton** [comment]
Updated with allowParens avoid

File: site/examples/markdown-preview.tsx [hunk 1]
```diff
@@ -70,38 +70,38 @@ const Leaf = ({ attributes, children, leaf }) => {
...
+          font-size: 20px;
+          margin: 20px 0 10px 0;
+        `}
         ${leaf.list &&
-          css`
-            padding-left: 10px;
-            font-size: 20px;
-            line-height: 10px;
-          `}
+        css`
```
Comment (timbuckley) [inline site/examples/markdown-preview.tsx:80]: This indentation is a little odd, is this all new-prettier's doing? One would normally think the second operand of a conjunction would be indented relative to the previous if it must be on a new line...
What if you try to put `css` immediately after the `&&` like so?
```
${leaf.title && css`
```
Does that change the indentation a bit? I

Comment (jason-krypton) [reply site/examples/markdown-preview.tsx:80]: All code changes are from running `yarn lint --fix`. I can try manually changing the line manually and re-running, but will be a bit before I update.

**Comment by jason-krypton** [comment]
I tried to manually change back the whitespace change around css that you called out @timbuckley but then lint will complain.

**Pull request closed without merging**